kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/SaslAuthenticator.java

```diff
@@ -324,12 +324,15 @@ private static void sendKafkaResponse(ChannelHandlerContext ctx,
         });
     }
 
-    private static ByteBuf sizePrefixed(ByteBuffer buffer) {
+    public static ByteBuf sizePrefixed(ByteBuffer buffer) {
```

Comment:
We can add annotation `@VisibleForTesting`, because this method should be private, change to public just for unit test.

Reply:
@Demogorgon314 Good suggestion. I have addressed your comment. Thanks.